config/manager/manager.yaml

```diff
@@ -60,9 +61,22 @@ spec:
             port: 8081
           initialDelaySeconds: 5
           periodSeconds: 10
+        resources:
+          limits:
+            cpu: 1000m
+            memory: 2000Mi
+          requests:
+            cpu: 1000m
+            memory: 2000Mi
+        volumeMounts:
+          - mountPath: /opt/splunk/appframework/
+            name: app-staging
       serviceAccountName: controller-manager
       volumes:
       - configMap:
           name: config
         name: splunk-operator-config
+      - name:  app-staging
+        persistentVolumeClaim:
+          claimName: tmp-app-download
```

Comment:
Nit: Can we remove the `tmp` word, considering it is PVC?

Reply:
fixed